node-api-postgres/queries.js

```diff
@@ -56,7 +56,28 @@ const authenticateUser = (request, response) => {
     });
 }
 
+const updateProfile = (request, response) => {
+    const { body } = request;
+    const name = body.name;
+    const email = body.email;
+
+    pool.query('UPDATE users SET name=$1, email=$2', [name, email], (error, results) => {
```

Comment:
prob gonna need to change this endpoint to check for newemail/old email
i think you are currently working on this so once it is changed i can look at it again